sauron/rule_engine.py

```diff
@@ -0,0 +1,5 @@
+from .engine import Engine
+
+
+class RuleEngine(Engine):
+    ...
```

Comment:
? this should ne changed to `pass` i guess

Reply:
This will be implemented soon... the three dots is the same as pass afaik